README.md

```diff
@@ -9,6 +9,40 @@ From time to time there is a need to merge the changes from the original repo in
 | --- | --- |
 |**Original corresponding commit hash:**| `d50aef95520df4216c638495a6049125c00742cb` |
 
+## Update Guide
```

Comment:
Add a phrase explaining what kind of update are you referring to and the whole purpose of this chapter.

Reply:
Will do. Fwiw, it's a subheading.